grow/rendering/rendered_document.py

```diff
@@ -23,6 +23,13 @@ def _get_tmp_filename(self):
         """Temp filename if has temporary dir."""
         return os.path.join(self.tmp_dir, self.hash)
 
+    @property
+    def filename(self):
```

Comment:
`filename` always feels like the file's basename to me, e.g. `index.html`. maybe use `local_filepath` or `local_file_path` instead?

Reply:
Good point, I've switched it to `file_path` instead.